CHANGELOG.md

```diff
@@ -4,20 +4,32 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [0.5.2] - 2022-08-24
```

Comment:
Is the date here intentional? Should it be 2022-09-14? It's fine if you are trying to indicate when this work was done, just asking.

Reply:
oh... I made it earlier... I will fix it now

Reply:
fixed